README.md

```diff
@@ -1,14 +1,27 @@
 # HarbourBridge: Turnkey Postgres-to-Spanner Evaluation
-[![cloudspannerecosystem](https://circleci.com/gh/cloudspannerecosystem/harbourbridge.svg?style=svg)](https://circleci.com/gh/cloudspannerecosystem/harbourbridge)
```

Comment:
You have removed this again. I assume from the second commit this is not intended